pkg/instance/updates.go

```diff
@@ -237,12 +250,26 @@ func filterSlug(slug string, slugs []string) bool {
 	return true
 }
 
-func createInstaller(inst *Instance, registries []*url.URL, man apps.Manifest) (*apps.Installer, error) {
+func createInstaller(inst *Instance, registries []*url.URL, man apps.Manifest, opts *UpdatesOptions) (*apps.Installer, error) {
+	var sourceURL string
+	if opts.ForceRegistry {
+		originalSourceURL, err := url.Parse(man.Source())
+		if err == nil && originalSourceURL.Scheme == "git" {
```

Comment:
`git` is enough? Don't we have some `git+ssh` or `ssh+git` for repositories on our gitlab?

Reply:
you're right, `git` is not enough, i missed that